config-sample.yml

```diff
@@ -15,12 +15,14 @@
 
 development:
   password:
-  theme_id: 
+  theme_id:
   store: your-shop.myshopify.com
   ignore_files:
+    #- settings_data.json # Uncomment this line to avoid resetting theme settings
 
 production:
   password:
   theme_id: "live"
   store: your-shop.myshopify.com
-  ignore_files:
\ No newline at end of file
+  ignore_files:
+    #- settings_data.json # Uncomment this line to avoid resetting theme settings
```

Comment:
Verdict is: merging as comment for now and going to do a couple more tests as a new issue.